webpack/JobWizard/JobWizard.scss

```diff
@@ -0,0 +1,74 @@
+.job-wizard {
+
+  .wizard-title {
+    margin-bottom: 25px;
+  }
+
+  .pf-c-wizard__main {
+    overflow: visible;
+    z-index: calc(
```

Comment:
Need to change `.pf-c-wizard__nav` z-index to be bigger than this so the menu in small will be visible